ICAFIX/hcp_fix_multi_run

```diff
@@ -198,18 +246,9 @@ log_Msg "Showing FSL version"
 fsl_version_get fsl_ver
 log_Msg "FSL version: ${fsl_ver}"
 
-# Show specific FIX version, if available
-if [ -f ${FSL_FIXDIR}/fixversion ]; then
-	fixversion=$(cat ${FSL_FIXDIR}/fixversion )
-	log_Msg "FIX version: $fixversion"
-fi
```

Comment:
Why remove altogether this reporting of `fix` version, rather than preserve it inside a `if(legacyFix)` conditional?

Reply:
Seems reasonable.

Reply:
Look up to line 218, it is still there:

https://github.com/Washington-University/HCPpipelines/blob/2297525b62590627a1ec8a22208f5d7523cc3f8e/ICAFIX/hcp_fix_multi_run#L218